src/main/java/appeng/server/Commands.java

```diff
@@ -22,12 +22,14 @@
 import appeng.server.subcommands.ChunkLogger;
 import appeng.server.subcommands.Supporters;
 import appeng.server.subcommands.Profile;
+import appeng.server.subcommands.ToggleFullAccess;
 
 public enum Commands
 {
 	Chunklogger( 4, new ChunkLogger() ),
 	Supporters( 0, new Supporters() ),
-	profile(2, new Profile() );
+	profile(2, new Profile() ),
+	toggleFullAccess(2, new ToggleFullAccess());
```

Comment:
Didn't boubou ask for access level 4?

Reply:
I understood it so that this command should work kind of like `su` for any OP. @boubou19 ?

Reply:
Using 4 seems rather excessive. Given 3 gives the ability to ban, kick etc it should probably be under that at a maximum.